apps/myaccount/src/pages/account-security.tsx

```diff
@@ -109,8 +156,20 @@ const AccountSecurityPage = (props: RouteProps): ReactElement => {
                             </Grid.Column>
                         </Grid.Row>
                     ) : null }
-
-                { !isReadOnlyUser &&
+                { /* Create password section temporarily commented until feature is planned. */ }
```

Comment:
Shall we create a git issue to track this and link it in the doc comment?

Reply:
I will get this issue created.